ocs_ci/ocs/resources/mcg.py

```diff
@@ -20,7 +22,7 @@ def __init__(self):
         """
         Constructor for the MCG class
         """
-        ocp_obj = OCP(kind='noobaa', namespace='noobaa')
+        ocp_obj = OCP(kind='noobaa', namespace=defaults.ROOK_CLUSTER_NAMESPACE)
```

Comment:
As part of [PR 652](https://github.com/red-hat-storage/ocs-ci/pull/652), both this and `secret_ocp_obj`'s namespaces are being changed to 'self.namespace`, which is defined as `config.ENV_DATA['cluster_namespace']`.

Reply:
changed